components/Navigation/index.js

```diff
@@ -0,0 +1,50 @@
+import Link from "next/link";
+import styled from "styled-components";
+import HomeSvg from "@/public/home.svg";
+
+export default function Navigation() {
+  return (
+    <StyledNavigation>
+      <StyledList>
+        <li>
+          <StyledLink href="/">
+            <StyledSvg />
+          </StyledLink>
+        </li>
+      </StyledList>
+    </StyledNavigation>
+  );
+}
+const StyledNavigation = styled.nav`
+  width: 100%;
+  margin-bottom: auto;
+  background-color: var(--primary-color);
+  color: var(--neutral-color);
+  bottom: 0;
+  color: white;
+  padding: 15px 0;
+  position: fixed;
+  bottom: 0;
+  padding: 15px;
+  display: grid;
+  z-index: 1;
+`;
+
+const StyledList = styled.ul`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: center;
+`;
+
+const StyledLink = styled(Link)`
+  color: white;
```

Comment:
HEX codes are generally preferred over color names like `white`, because they provide a precise definition of a color - in case of the color white is of course debatable 😁

```suggestion
  color: #fff;
```

Reply:
Good point! We changed it and used our variable color from the GlobalStyles which is defined by a hex code.